modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/raft/snapshot/incoming/IncomingSnapshotCopier.java

```diff
@@ -133,12 +137,13 @@ public class IncomingSnapshotCopier extends SnapshotCopier {
     public IncomingSnapshotCopier(
             PartitionSnapshotStorage partitionSnapshotStorage,
             SnapshotUri snapshotUri,
-            Executor executor,
+            ExecutorService executor,
```

Comment:
You changed `Executor` to `ExecutorService` because `IgniteThrottledLoggerImpl` wants the latter, but in reality `Executor` is enough in the logger. I suggest to change `ExecutorService` to `Executor` in the logger and revert this part of the changes.

Reply:
Nice catch, didn't notice that